apps/chat-e2e/src/ui/webElements/marketplace/marketplaceContainer.ts

```diff
@@ -22,4 +24,8 @@ export class MarketplaceContainer extends BaseLayoutContainer {
     }
     return this.marketplaceSidebar;
   }
+
+  getChatLoader(): ChatLoader {
```

Comment:
as I see, the current loader should not refer to ChatLoader class and must be initialized in a different way:

`public loader = this.getChildElementBySelector(ChatSelectors.messageSpinner);`